### PR TITLE
Us1

### PR DESCRIPTION
**Pull request by FranckWebPro**

### CODE REVIEW

File: client/src/components/CocktailCard.jsx
Comment (Yoann-Fortin) [inline client/src/components/CocktailCard.jsx:1]: A ghost component ? Where is this logic ?

File: client/src/components/OrangeButton.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import PropTypes from "prop-types";
+
+function OrangeButton({ season }) {
+  return (
+    <a href={`${season.toLowerCase()}`} className="orange-button">
+      <p className="season">{season}</p>
```
Comment (Yoann-Fortin) [inline client/src/components/OrangeButton.jsx]: Isn't possible, check W3C validator

File: client/src/components/OrangeButton.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import PropTypes from "prop-types";
+
+function OrangeButton({ season }) {
```
Comment (Yoann-Fortin) [inline client/src/components/OrangeButton.jsx]: Name isn't the real call to action

File: client/src/styles/CocktailsSection.css
Comment (Yoann-Fortin) [inline client/src/styles/CocktailsSection.css:1]: No empty file please

File: client/src/styles/SeasonCard.css [hunk 3]
```diff
@@ -0,0 +1,153 @@
+:root {
```
Comment (Yoann-Fortin) [inline client/src/styles/SeasonCard.css]: is already defined

File: client/src/styles/SeasonCard.css [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  font-size: 25px;
+  color: var(--text-color);
+}
+
+.card {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+  min-height: calc(50vh);
```
Comment (Yoann-Fortin) [inline client/src/styles/SeasonCard.css]: calc is overkill

File: client/src/styles/OrangeButton.css
Comment (Yoann-Fortin) [inline client/src/styles/OrangeButton.css:1]: No empty file please

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/src/components/OrangeButton.jsx (hunk 2 above)
Comment (Yoann-Fortin) [reply client/src/components/OrangeButton.jsx]: This isn't change yet

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

File: client/src/components/OrangeLink.jsx [hunk 5]
```diff
@@ -1,15 +1,26 @@
 import PropTypes from "prop-types";
 
-function OrangeLink({ season }) {
+function OrangeLink({ season, cocktailName, id }) {
   return (
-    <a href={`${season.toLowerCase()}`} className="orange-button">
-      {season}
+    <a
```
Comment (Yoann-Fortin) [inline client/src/components/OrangeLink.jsx]: With Link and generic props please

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/src/components/CocktailPage.jsx
Comment (Yoann-Fortin) [inline client/src/components/CocktailPage.jsx:1]: Commented code is not possible

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

File: client/src/pages/CocktailPage.jsx [hunk 6]
```diff
@@ -1,5 +1,52 @@
-function CocktailPage() {
-  return <p>dsfh</p>;
```
Comment (JulienChevreauPro) [inline client/src/pages/CocktailPage.jsx:2]: what is dsfh?

File: client/src/pages/CocktailPage.jsx
Comment (JulienChevreauPro) [inline client/src/pages/CocktailPage.jsx:1]: comments

**Review by JulienChevreauPro — CHANGES_REQUESTED** [review]

**Review by JulienChevreauPro — APPROVED** [review]

**Pull request merged**